thorlcr/activities/thdiskbase.cpp

```diff
@@ -302,7 +302,15 @@ void CWriteMasterBase::publish()
                             queryJob().fireException(e2);
                         }
                     }
-                    partDesc->queryProperties().setPropInt64("@size", 0);
+                    IPropertyTree &props = partDesc->queryProperties();
+                    StringBuffer timeStr;
+                    modifiedTime.setNow();
```

Comment:
Should modified time change ?

Reply:
it really shouldn't matter and very likely to be within same second, so same time,
but have changed it to get the actual time of the part created and use that.

Please re-review